src/style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-left: 3px;
+}
+
+.head {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+  text-align: left;
+  margin-top: 40px;
+
+  margin-bottom: 0;
+}
+
+.parag {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  text-align: left;
+  color: #616070;
+  margin-top: 4px;
+  
+  margin-bottom: 16px;
+}
+
+.mark {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
You need to implement appended instruction here.

- Reuse `stars` block from [Stars task](https://github.com/mate-academy/layout_stars)
and keep it operational. If you change `stars--4` modifier to `stars--2` or
other - number of selected stars should change correctly.